src/docs/asciidoc/reference_doc/ui_api.adoc

```diff
@@ -54,6 +55,18 @@ Obtain all entities as an array :
 const entities = opfab.users.entities.getAllEntities();
 ```
 
+== BusinessConfig API
+
+=== Function registerFunctionToGetTags
+
+It is possible to register at startup the function to be used to retrieve the list of tags:
+
+```
+opfab.businessconfig.registerFunctionToGetTags(function);
+```
+
+An example of usage can be found in the file
+https://github.com/opfab/operatorfabric-core/tree/master/config/docker/externalJs/loadTags.js[config/docker/externalJs/loadTags.js].
```

Comment:
Explain that if you return undefined in the registered function the tag list is taken from the web-ui.json config